lib/run.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 
+source lib/print.sh
+
 run() {
   echo -e "\x1B[1;34m==\$ $*\x1B[0m"
   eval "$*"
 }
+
+run_with_retry() {
+  if ! run "$*" && ask_Yn "Something went wrong. Try again?"; then
```

Comment:
In the case of Homebrew failing to install apps that have already been installed (by other means), it's not really that something went wrong so much as Homebrew just had less work to do. I guess devs should be able to deduce from the output what's happened and say no, but is it worth adding a sentence to step five of [How do I use this?](https://github.com/dxw/local-env#how-do-i-use-this) along the lines of this?

> If you see the prompt `Something went wrong. Try again?` after a line like `Homebrew Bundle failed! 2 Brewfile dependencies failed to install.`, it means Homebrew was unable to install some apps you already had installed - this is fine, and you can safely type "n" to continue.

Reply:
I'd rather include that advice inline than in a readme nobody will read. Maybe the answer here is to remove the "Something went wrong" part actually? And just rely on the thing you're running to report its status.

Reply:
Sounds good!